.gitignore

```diff
@@ -1 +1,4 @@
 fake-gcs-server
+
+# editor and IDE
+.idea
```

Comment:
Can you remove this? For editor specific files, I recommend something like a global ignore file.

Reply:
Sure. Done.